app/main.py

```diff
@@ -1,25 +1,71 @@
 from abc import ABC
+from typing import Any
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: type, name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: object, owner: type) -> object:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: object, value: Any) -> None:
```

Comment:
Annotations for instance and owner are wrong

Reply:
I'm trying to understand my mistake in the type annotations for instance and owner, but in my opinion, the types are correct

Reply:
Instance is not just object, it's a specific object of class IntegerRange.  `Type[IntegerRange]`,

Owner is an object of Validation class: `Type[SlideLimitationValidator]`

Reply:
I can not add Type[IntegerRange] annotation in class IntegerRange

Reply:
You can add this type annotation if you import this: `from __future__ import annotations` at the beginning of the file. 
<img width="957" alt="image" src="https://github.com/mate-academy/py-aquapark/assets/37514514/51421ac5-cee2-4632-845c-140e6476bd27">

Reply:
Got it, sorry about that